pkg/pillar/types/dpc.go

```diff
@@ -401,6 +402,27 @@ func (config *DevicePortConfig) DoSanitize(log *base.LogObject, args DPCSanitize
 				config.Ports[i].IsL3Port = true
 			}
 		}
+
+		sanitizeNTPServers := func(servers *[]string) []string {
+			var sanitizedNTPServers []string
```

Comment:
It is somewhat confusing to have this local variable in the func have the same name as the local variable in the outer scope. Maybe this one could be named "res" or something different?

Reply:
@eriknordmark fixed